astroid/node_classes.py

```diff
@@ -2583,6 +2586,7 @@ def __init__(self, value, lineno=None, col_offset=None, parent=None):
         :type parent: NodeNG or None
         """
         self.value = value
+        self.kind = kind
         """The value that the constant represents.
 
         :type: object
```

Comment:
Can you remove this comment? It seems to belong to `self.value` but that is already mentioned in the function docstring.